securedrop_client/gui/widgets.py

```diff
@@ -132,27 +145,29 @@ class MainView(QWidget):
     def __init__(self, parent):
         super().__init__(parent)
         self.layout = QHBoxLayout(self)
+        self.layout.setContentsMargins(0, 0, 0, 0)
         self.setLayout(self.layout)
 
         left_column = QWidget(parent=self)
         left_layout = QVBoxLayout()
+        left_layout.setContentsMargins(0, 0, 0, 0)
         left_column.setLayout(left_layout)
 
-        self.status = QLabel(_('Waiting to refresh...'))
         self.error_status = QLabel('')
         self.error_status.setObjectName('error_label')
-
-        left_layout.addWidget(self.status)
-        left_layout.addWidget(self.error_status)
+        # TODO: move this to the main window QStatusBar
```

Comment:
We shouldn't include TODOs in the code. They should be tickets. Likewise, we shouldn't include commented out code / dead code.